core/src/test/java/io/questdb/test/griffin/O3PartitionPurgeTest.java

```diff
@@ -250,7 +250,7 @@ public void testManyReadersOpenClosedDescSparse() throws Exception {
 
     @Test
     public void testManyTablesFuzzTest() throws Exception {
-        Rnd rnd = TestUtils.generateRandom(LOG);
+        Rnd rnd = TestUtils.generateRandom(LOG, 127668244828333L, 1687527306083L);
```

Comment:
Looks like a debug leftover. Is that the case?

Reply:
removed